app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetUpView.kt

```diff
@@ -45,8 +46,8 @@ class MeetUpView : AppCompatActivity() {
 
         val meetupId = intent.getSerializableExtra(MEETUP_SHOWN) as String
         viewModel.loadMeetUp(meetupId)
-        //button to show the list of users participating in this meetUp
-        val button = findViewById<Button>(R.id.show_profile_list_button)
+        //button (image for design purpuses) to show the list of users participating in this meetUp
+        val button = findViewById<ImageView>(R.id.show_profile_list_button)
```

Comment:
Could we remove this and add the onClick in the XML file?

Reply:
I just kept the code as is. But in general @zaclegarssure said it's better to have them in the code, it allows him to integrate these into the app flow better. In the end it boils down to preference, since we didn't decide on what format to consistently use, both work.